src/views/Pages/Message/RegMessage.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import { Col, Container, Row } from 'reactstrap';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Col' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)